### ISSUE TITLE
Microsoft.CodeAnalysis.Common NuGet package is incompatible with VS NuGet extension

### ISSUE THREAD

**Issue by int19h** [dotnet/roslyn#12270]
VS 2015 Update 3 (but also observed in Update 2 prior to upgrade).

NuGet extension v3.4.4 (most recent stable) or v3.5.0-beta2 (most recent beta).

Repro steps:

1. Create a new Windows | Classic Desktop | Console Application project.
2. Right-click project -> Manage NuGet packages.
3. Find Microsoft.CodeAnalysis.Common in Browse, and install it.
4. Switch to the Installed tab in NuGet.

Result:

![image](https://cloud.githubusercontent.com/assets/1175142/16436545/7d9ceb04-3d53-11e6-8d58-0f41815cb255.png)

Full text of the error message that is displayed in the Output window is:
> The schema version of 'Microsoft.CodeAnalysis.Common' is incompatible with version 2.12.0.815 of NuGet. Please upgrade NuGet to the latest version from http://go.microsoft.com/fwlink/?LinkId=213942.

The version number in the error message seems to be coming from NuGet.Core.dll, which has the same version number regardless of which version of NuGet VS extension I install.

Looking at the .nuspec file for the package, it seems that it references a new schema:
```xml
<package xmlns="http://schemas.microsoft.com/packaging/2016/06/nuspec.xsd">
```

Note that this package is a dependency for Microsoft.AspNetCore.Mvc; so there seems to be a problem trying to use that in non-.NET Core projects in VS right now, despite it nominally being a supported platform.

**Comment by int19h** [comment]
I guess that can be interpreted as a bug on NuGet and/or VS in that they do not have a public version of the extension to handle the new schema. But then, shouldn't that version not be used until there's proper support for it in the tooling?

**Comment by tmat** [comment]
What version of Microsoft.CodeAnalysis.Common are you trying to install? Can you try clearing your nuget cache (```nuget locals all -Clear```) before restoring?

**Comment by int19h** [comment]
Version 1.3.0 - the most recent stable that's on the main NuGet repo (https://www.nuget.org/packages/Microsoft.CodeAnalysis.Common/).

I don't have nuget command line installed at all, so I'm not sure what the equivalent cache purge command is in my case. However, I have deleted AppData\Local\NuGet entirely, but it didn't help.

**Comment by rrelyea** [comment]
Talk to chuck stoner. That package you have posted has a nuspec xmlns uri that should never have been published. Please repack that package and ship a new version.

**Comment by int19h** [comment]
It turned out to be a cache issue. At some brief period of time, the 1.3.0 package did have that xmlns, but it was updated in-place with the correct version. I just happened to try to install it during that period, and it got cached; so subsequent installs just re-installed that broken cached version.

The fix is to get [nuget.exe](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe), and then run:
```sh
nuget locals all -Clear
```
Then uninstall and reinstall the package.
